pycbc/psd/analytical_space.py

```diff
@@ -157,18 +355,20 @@ def analytical_psd_lisa_tdi_1p5_XYZ(length, delta_f, low_freq_cutoff,
     acc_noise_level = np.float64(acc_noise_level)
     oms_noise_level = np.float64(oms_noise_level)
     fr = np.linspace(low_freq_cutoff, (length-1)*2*delta_f, length)
-    s_acc_nu, s_oms_nu = lisa_psd_components(
-                          fr, acc_noise_level, oms_noise_level)
-    omega_len = omega_length(fr, len_arm)
-    psd = 16*(np.sin(omega_len))**2 * (s_oms_nu+s_acc_nu*(3+np.cos(omega_len)))
+    s_acc_nu = _psd_acc_noise(fr, acc_noise_level)
+    s_oms_nu = _psd_oms_noise(fr, oms_noise_level)
```

Comment:
These lines indicate that psd_detector_acc/oms_noise functions will not be used, especially after you apply changes for TianQin components. This issue also shows in PSD functions for other channels.

Reply:
@HumphreyWang They will be used in LISA/Taiji case, because those two share the same frequency dependency in PSD of acc/oms noise. For TianQin, I'm not using those, I just code in TianQin's functions.

Reply:
Right, but for funcs like `analytical_psd_tianqin_tdi_1p5_XYZ`, now you are still calling **this** function, so the acc/oms parameters will pass to `_psd_acc/oms_noise` instead of `psd_tianqin_acc/oms_noise` you edited in last commit.

Reply:
OK. I will update them.

Reply:
@HumphreyWang Fixed
![image](https://github.com/gwastro/pycbc/assets/6574526/5fafbc57-c75d-48bf-9d87-6d779f9911a0)
![image](https://github.com/gwastro/pycbc/assets/6574526/b9ae80cb-2239-4e1a-bfbb-979e180c90b2)
![image](https://github.com/gwastro/pycbc/assets/6574526/5bfc26eb-4437-4f4b-99b4-3cc0526071d4)
![image](https://github.com/gwastro/pycbc/assets/6574526/948d45eb-4a7e-4dd1-953a-7d6d4063f806)